app/assets/javascripts/components/student/components/cancel_modal.es6.jsx

```diff
@@ -0,0 +1,97 @@
+class CancelModal extends React.Component {
+
+  constructor() {
+    super();
+    this.state = {
+      showNextScreen: false,
+    };
+  }
+
+  static get propTypes() {
+    return {
+      handleClose: React.PropTypes.func.isRequired,
+      fetchUpcomingLessons: React.PropTypes.func.isRequired,
+      lesson: React.PropTypes.object.isRequired,
+    };
+  }
+
+  handleNext() {
+    this.setState({ showNextScreen: true });
+  }
+
+
+
+  handleCancelLesson() {
```

Comment:
let's just call this handleConfirmClick so it doesn't conflict with handleCancelLesson